apps/wakunode2/config.nim

```diff
@@ -185,6 +186,10 @@ type
       defaultValue: ""
       name: "rln-relay-cred-password" }: string
 
+    extMultiAddrs* {.
+      desc: "External multiaddresses to advertise to the network. Argument may be repeated."
+      name: "ext-multiaddr" }: seq[string]
```

Comment:
I think this option should be located after the `nat` option. Since it is related to "transport".

Reply:
Addressed in [b21f1ed](https://github.com/waku-org/nwaku/pull/1509/commits/b21f1ed0f4126e1166c234a9b9a0893f8dc84121). I thought I would colocate the multiaddrs options together (with staticnode)